<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1147</width>
    <height>753</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="MyComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>720</y>
     <width>96</width>
     <height>28</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1141</width>
     <height>711</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Seed Phrase</string>
    </attribute>
    <widget class="QLineEdit" name="lineEdit_pub_c">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>310</y>
       <width>891</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Liberation Mono</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0x3AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Derivation path:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>360</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Monospace</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Address:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Mnemonic:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_k">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>260</y>
       <width>891</width>
       <height>27</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Liberation Mono</family>
       <pointsize>13</pointsize>
       <italic>false</italic>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_address">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>360</y>
       <width>891</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Liberation Mono</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_derivation_path_index">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>180</y>
       <width>891</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Index (x):</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>PrivKey:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_derivation_path">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>140</y>
       <width>891</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>m/44'/60'/0'/0/x</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_mnc">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>1001</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Liberation Mono</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="plainText">
      <string>heavy kind unusual say discover cram false dance cement pottery churn inside rocket quantum image achieve museum current betray tide subject seat sting slight</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>PubKey:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_bip39_pwd">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>220</y>
       <width>891</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>BIP39 password:</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_12_words">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>100</y>
       <width>127</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>12-words</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_24_words">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>100</y>
       <width>127</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>24-words</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Decrypt Tx</string>
    </attribute>
    <widget class="QPlainTextEdit" name="plainTextEdit_raw_tx">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>1051</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Liberation Mono</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="plainText">
      <string>0xf86c0d8503291dfe6282520894912fd21d7a69678227fe6d08c64222db41477ba08802c558495caa80f08025a06b89e06433a6e7a0401051560b112a0bb6537c0ba229218036ceea43273eaa18a05663272729cda3a852e50bc4cae7cd0bb2ae751c4154978a924682f20a58fb63</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Raw Tx:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Nonce:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_nonce">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>220</y>
       <width>890</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Liberation Mono</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_max_priority_fee_per_gas">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>250</y>
       <width>890</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Liberation Mono</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>max_priority_fee_per_gas:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_gaslimit">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>340</y>
       <width>890</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Liberation Mono</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Gas Limit:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_to">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>370</y>
       <width>890</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Liberation Mono</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>370</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>To:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_eth">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>400</y>
       <width>890</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Liberation Mono</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>400</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Eth:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_data">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>430</y>
       <width>890</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Liberation Mono</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>430</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Data:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_v">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>490</y>
       <width>890</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Liberation Mono</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>490</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>v:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_r">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>520</y>
       <width>891</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Liberation Mono</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>520</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>r:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_s">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>550</y>
       <width>891</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Liberation Mono</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>550</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>s:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_tx_hash">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>140</y>
       <width>890</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Liberation Mono</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Tx Hash:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_chainId">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>190</y>
       <width>890</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Liberation Mono</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ChainId:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_max_fee_per_gas">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>280</y>
       <width>890</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Liberation Mono</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>max_fee_per_gas:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_gasprice">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>310</y>
       <width>890</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Liberation Mono</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Gas Price:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_access_list">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>460</y>
       <width>890</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Liberation Mono</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>460</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>AccessList:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>600</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>PubKey(Ecrecover):</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_from_pubkey">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>600</y>
       <width>891</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Liberation Mono</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0x3AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_from_address">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>630</y>
       <width>890</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Liberation Mono</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>630</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Address(Ecrecover):</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_pre_eip155">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>100</y>
       <width>141</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Pre-EIP155</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_eip155">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>100</y>
       <width>127</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>EIP155</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_eip2930">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>100</y>
       <width>127</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>EIP2930</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_eip1559">
     <property name="geometry">
      <rect>
       <x>1010</x>
       <y>100</y>
       <width>127</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>EIP1559</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_invalid_tx">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>100</y>
       <width>141</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Invalid Tx</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyComboBox</class>
   <extends>QComboBox</extends>
   <header>mycombobox.h</header>
   <slots>
    <slot>slot1()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
